2024-04-18T17:18:42 {"cpu": 0.470703125, "cpu_total": 8, "cpu_usage": 0.058837890625, "memory": 8085.0, "memory_total": 14964.0, "memory_usage": 0.5402967121090617, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460722.730657, "uuid": "8bb619838a10467b9cba31150702ba7b"}
2024-04-18T17:18:43 {"cpu": 0.470703125, "cpu_total": 8, "cpu_usage": 0.058837890625, "memory": 8115.0, "memory_total": 14964.0, "memory_usage": 0.5423015236567763, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460723.004528, "uuid": "5feec1d5803f43789e072c0a5e770976"}
2024-04-18T17:18:43 {"block_run_id": 7, "block_uuid": "get_news", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 7: pipeline tengrinews_get_post_yesterday block get_news", "timestamp": 1713460723.091415, "uuid": "1a991585d2c94059b9937df9fb97f985"}
2024-04-18T17:18:52 {"cpu": 0.470703125, "cpu_total": 8, "cpu_usage": 0.058837890625, "memory": 8201.0, "memory_total": 14964.0, "memory_usage": 0.5480486500935579, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460732.614375, "uuid": "6a2a526358dd4f29a0a804eba7de32ef"}
2024-04-18T17:19:03 {"cpu": 0.4765625, "cpu_total": 8, "cpu_usage": 0.0595703125, "memory": 8205.0, "memory_total": 14964.0, "memory_usage": 0.5483159582999199, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460743.145502, "uuid": "bd6e5bbacc7444b98c2b4ad56381f025"}
2024-04-18T17:19:12 {"cpu": 0.49853515625, "cpu_total": 8, "cpu_usage": 0.06231689453125, "memory": 8194.0, "memory_total": 14964.0, "memory_usage": 0.5475808607324245, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460752.63217, "uuid": "d4988d562a694d8592840bc200e12056"}
2024-04-18T17:19:23 {"cpu": 0.51513671875, "cpu_total": 8, "cpu_usage": 0.06439208984375, "memory": 8200.0, "memory_total": 14964.0, "memory_usage": 0.5479818230419674, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460763.229532, "uuid": "980c1685701f44ad8af1e52a15e462e3"}
2024-04-18T17:19:32 {"cpu": 0.5263671875, "cpu_total": 8, "cpu_usage": 0.0657958984375, "memory": 8201.0, "memory_total": 14964.0, "memory_usage": 0.5480486500935579, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460772.817217, "uuid": "502c058718594423b827bfa3d8efb996"}
2024-04-18T17:19:43 {"cpu": 0.53759765625, "cpu_total": 8, "cpu_usage": 0.06719970703125, "memory": 8207.0, "memory_total": 14964.0, "memory_usage": 0.5484496124031008, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460783.412456, "uuid": "30fdec665e12496eba525bbdcd46d14e"}
2024-04-18T17:19:49 {"block_run_id": 7, "block_uuid": "get_news", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "BlockRun 7 (block_uuid: get_news) completes.", "timestamp": 1713460789.708839, "uuid": "3a1ece6623c2483e81f4dc19864b0caa"}
2024-04-18T17:19:53 {"cpu": 0.54248046875, "cpu_total": 8, "cpu_usage": 0.06781005859375, "memory": 8071.0, "memory_total": 14964.0, "memory_usage": 0.5393611333867949, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460793.035047, "uuid": "eaa06f5a175d4585907245e162074756"}
2024-04-18T17:19:53 {"block_run_id": 8, "block_uuid": "transform_news", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 8: pipeline tengrinews_get_post_yesterday block transform_news", "timestamp": 1713460793.580291, "uuid": "4a5801ff6f2c4b90ba597e5647aa034c"}
2024-04-18T17:19:54 {"block_run_id": 8, "block_uuid": "transform_news", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "BlockRun 8 (block_uuid: transform_news) completes.", "timestamp": 1713460794.493653, "uuid": "9122a02c5821413994514ef0270ef0e6"}
2024-04-18T17:20:02 {"cpu": 0.552734375, "cpu_total": 8, "cpu_usage": 0.069091796875, "memory": 8077.0, "memory_total": 14964.0, "memory_usage": 0.5397620956963379, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460802.769512, "uuid": "66cfcf722f0e4950af1aeb8586822cd7"}
2024-04-18T17:20:03 {"block_run_id": 9, "block_uuid": "load_news_to_gcs", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Execute PipelineRun 3, BlockRun 9: pipeline tengrinews_get_post_yesterday block load_news_to_gcs", "timestamp": 1713460803.241349, "uuid": "834bcfcceb454c6486373cd0c7aa0f4f"}
2024-04-18T17:20:04 {"block_run_id": 9, "block_uuid": "load_news_to_gcs", "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "EXCEPTION", "message": "BlockRun 9 (block_uuid: load_news_to_gcs) failed.", "timestamp": 1713460804.966144, "uuid": "6937be295834485e8688dee17e4bb4f4", "error": ["Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 613, in execute\n    result = __execute_with_retry()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 54, in retry_func\n    raise e\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 38, in retry_func\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 588, in __execute_with_retry\n    return self._execute(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 1077, in _execute\n    result = self.block.execute_sync(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1312, in execute_sync\n    raise err\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1221, in execute_sync\n    output = self.execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1527, in execute_block\n    outputs = self._execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1683, in _execute_block\n    outputs = self.execute_block_function(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1722, in execute_block_function\n    output = block_function_updated(*input_vars, **global_vars)\n  File \"<string>\", line 27, in export_data\nAttributeError: 'pyarrow._gcsfs.GcsFileSystem' object has no attribute 'exists'\n"], "error_stack": [["  File \"/usr/local/bin/mage\", line 8, in <module>\n    sys.exit(app())\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 311, in __call__\n    return get_command(self)(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1130, in __call__\n    return self.main(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 778, in main\n    return _main(\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 216, in _main\n    rv = self.invoke(ctx)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1657, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1404, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 760, in invoke\n    return __callback(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 683, in wrapper\n    return callback(**use_params)  # type: ignore\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/cli/main.py\", line 163, in start\n    start_server(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/server.py\", line 659, in start_server\n    scheduler_manager.start_scheduler()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 87, in start_scheduler\n    proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 50, in run_scheduler\n    LoopTimeTrigger().start()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/loop_time_trigger.py\", line 14, in start\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/time_trigger.py\", line 11, in run\n    schedule_all()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1603, in schedule_all\n    PipelineScheduler(r).schedule()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 149, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 319, in schedule\n    self.__schedule_blocks(block_runs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 584, in __schedule_blocks\n    job_manager.add_job(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/job_manager.py\", line 27, in add_job\n    self.queue.enqueue(job_id, target, *args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 108, in enqueue\n    self.start_worker_pool()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 183, in start_worker_pool\n    self.worker_pool_proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 288, in poll_job_and_execute\n    worker.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/newrelic/api/background_task.py\", line 117, in wrapper\n    return wrapped(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 253, in run\n    start_session_and_run(args[1], *args[2], **args[3])\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1128, in run_block\n    return ExecutorFactory.get_block_executor(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 641, in execute\n    on_failure(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 149, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 415, in on_block_failure\n    self.logger.exception(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 30, in exception\n    self.__send_message('exception', message, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 59, in __send_message\n    data['error_stack'] = traceback.format_stack(),\n"]], "error_stacktrace": ["'pyarrow._gcsfs.GcsFileSystem' object has no attribute 'exists'"]}
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 613, in execute
    result = __execute_with_retry()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 54, in retry_func
    raise e
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 38, in retry_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 588, in __execute_with_retry
    return self._execute(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 1077, in _execute
    result = self.block.execute_sync(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1312, in execute_sync
    raise err
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1221, in execute_sync
    output = self.execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1527, in execute_block
    outputs = self._execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1683, in _execute_block
    outputs = self.execute_block_function(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1722, in execute_block_function
    output = block_function_updated(*input_vars, **global_vars)
  File "<string>", line 27, in export_data
AttributeError: 'pyarrow._gcsfs.GcsFileSystem' object has no attribute 'exists'
2024-04-18T17:20:12 {"cpu": 0.5517578125, "cpu_total": 8, "cpu_usage": 0.0689697265625, "memory": 8078.0, "memory_total": 14964.0, "memory_usage": 0.5398289227479284, "pipeline_run_id": 3, "pipeline_schedule_id": 2, "pipeline_uuid": "tengrinews_get_post_yesterday", "hostname": "1c79bfcd593c", "level": "INFO", "message": "Pipeline tengrinews_get_post_yesterday for run 3 in schedule 2 is alive.", "timestamp": 1713460812.503692, "uuid": "2f11c3967f7f4cd391191746adf62c67"}
